packages/web3-common/src/eth_execution_api.ts

```diff
@@ -123,9 +125,6 @@ export interface Block {
 	readonly hash: HexString32Bytes | null;
 }
 
-// https://github.com/ethereum/execution-apis/blob/main/src/schemas/filter.json#L59
-export type Topic = HexString256Bytes;
-
```

Comment:
Moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/src/types.ts#L23)